### DISCUSSION TITLE
Something is wrong when i touch the "explore"

### DISCUSSION THREAD

**Discussion by iwillmissu** [community/community#40085] [Mobile]
I do nothing but click the "explore".then it's wrong.

![Screenshot_2022-11-23-11-44-46-365_com.github.android.jpg](https://user-images.githubusercontent.com/61586527/204015278-b17c7829-1ab9-4f02-85c6-e1c810002728.jpg)![Screenshot_2022-11-22-23-16-15-799_com.github.android.jpg](https://user-images.githubusercontent.com/61586527/204015425-d856dbe3-9101-4807-a626-b23bc40eca5b.jpg)

**Comment by ossanna16** [comment]
Hi there @iwillmissu and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
